### ISSUE TITLE
Old version on F-Droid

### ISSUE THREAD

**Issue by bam80** [Phantast/smartnavi#21]
F-Droid has only 2.2.4f as latest version. However, 2.4.0 available from Google Play

**Comment by Phantast** [comment]
Yes, that's right. SmartNavi no longer supports F-Droid. Let me quickly point out why:
Providing a version for F-Droid is quite an effort because there are many Google libraries I normally use in SmartNavi especially Google Maps as Map Provider. I would only invest that effort again if there is a good reason. But to be honest the support from the community is not existing. I don't have ads or paid content because I hate such shit. Donations or just any kind of feedback are the only support I want. And there was nearly no feedback or support. The biggest bullshit is the rule that F-Droid versions are not allowed to track the usage (even anonymous) of the app without explicit user permission. So I am not even allowed to see if I invest effort and my free time for 10 or 100,000 users. So why should I do it, if nobody gives feedback and I am not allowed to see how many people use it.

**Comment by bam80** [comment]
Oh, sorry. Didn't see your reply.
Christian, I'm very frustrated to know you drop F-Droid. Only if I could help..
Personally for me using free programs from F-Droid, with it's rules is kind of moral choose. It's more importantly for me than technical benefits of using convenient proprietary libraries, etc.
Is proprietary Mapping API necessary for your project? I don't know much of this, but quick search gives quite a lot choices: https://www.programmableweb.com/news/top-10-mapping-apis-google-maps-microsoft-bing-maps-and-mapquest/analysis/2015/02/23.
Also, why are you against explicit user permission to track? Don't you think users would agree to share some statistics if you ask kindly? I wouldn't mind. I thinks there are plenty of possibilities to get it instead of implicit tracking. Sad you get bad luck with it.
Anyway, thank you a lot for your great project!
Hope you return to F-Droid, some day.. Good luck!
